tests/sles4sap/saptune/mr_test.pm

```diff
@@ -17,6 +17,20 @@ use version_utils 'is_sle';
 use strict;
 use warnings;
 
+sub reboot_wait {
+    my ($self) = @_;
+
+    $self->reboot;
+
+    # Wait for tuned to tune everything
+    my $timeout = 60 * get_var('TIMEOUT_SCALE', 1);
+    if (is_sle('>=15')) {
+        assert_script_run "timeout $timeout bash -c 'until tuned-adm verify >/dev/null ; do sleep 1 ; done'", $timeout;
+    } else {
+        sleep 60;
```

Comment:
Suggestion:
```
    my $timeout = 60;
￼    if (is_sle('>=15')) {
￼        assert_script_run "timeout $timeout bash -c 'until tuned-adm verify >/dev/null ; do sleep 1 ; done'", $timeout;
￼    } else {
￼        sleep bmwqemu::scale_timeout(60);
    }
```
With `TIMEOUT_SCALE` set, `assert_script_run` will already scale the timeout, so you just need to specifically scale it on the sleep. Not sure whether you'd also want to scale the timeout in the bash script.

Reply:
Fixed.